source/guides/getting-started/installing-cypress.md

```diff
@@ -288,3 +288,39 @@ CYPRESS_DOWNLOAD_MIRROR="https://www.example.com" cypress install
 ```
 
 Cypress will then attempt to download a binary with this format: `https://www.example.com/desktop/:version?platform=p`
+
+## Opt out of sending exception data to Cypress
```

Comment:
You should also mention that users can just set the environment variable when running Cypress - rather than exporting it.

```
CYPRESS_CRASH_REPORTS=0 cypress open
CYPRESS_CRASH_REPORTS=0 cypress run
```

Reply:
Yah, I doubt they would want to do this one-off though, right? Plus this is Linux specific.

Reply:
They might want to do that in CI. It's not linux specific - it's linux + OSX specific. Only windows is different.